ajax/libs/sql-formatter/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "sql-formatter",
+  "version": "2.3.1",
```

Comment:
Let's remove version field.
Because we add this lib via single package.json.
If you add version field, this means that you have to add version folder & add files from upstream under the version folder.
This may a little be hard for the beginner.
Thanks